bundle/Resources/config/services/cache.yaml

```diff
@@ -0,0 +1,3 @@
+services:
+    netgen_layouts.remote_media.cache.pool:
```

Comment:
Rather than using a synthetic service, why not use a `NullAdapter` by default and then set the correct one in the compiler pass?